nss-cohort-36/bangazon-client-virtual-magicians-1
# Sp shopping cart

# Description
User can now view all items in their Shopping Cart through `ShoppingCart.js` component.
Also added Home icon in `NavBar`

Closes PART OF: https://github.com/nss-cohort-36/bangazon-api-virtual_magicians/issues/2
## Type of change

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-shopping-cart`
4. If you have loaded fixtures from the API pull request, you should be able to see items in your shopping cart.

# Checklist:
